src/demos/namespaced_example /app.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```

Comment:
github UI displays a whitespace in the foldername, not sure if this breaks something in the future
<img width="458" alt="Screenshot 2022-01-27 at 16 21 15" src="https://user-images.githubusercontent.com/38131488/151388490-bc924887-5d7f-4a0f-8549-d4a1df942c14.png">

Reply:
good catch!